libnmstate/nm/connection.py

```diff
@@ -397,3 +397,11 @@ def get_device_active_connection(nm_device):
     if nm_device:
         active_conn = nm_device.get_active_connection()
     return active_conn
+
+
+def get_iface_name(active_connection):
```

Comment:
I think this one belongs to active_connection module.
You can even use the existing class: ActiveConnection
```
aconnection = ActiveConnection(ac)
aconnection.devname
```